src/components/UI/Dropdown/Dropdown.tsx

```diff
@@ -46,14 +46,16 @@ const Dropdown = <T,>({
     return selectedOption.label === option.label
   }
 
-  const selectImageUrl = isOpen ? openSelect : closedSelect
-
   return (
-    <div ref={ref} className={'relative' + ' ' + (className ?? '')}>
+    <div
+      id="dropdown"
+      ref={ref}
+      className={'relative text-L text-primary' + ' ' + (className ?? '')}
```

Comment:
I think it's better to use twerge here. It allows to override styles of the component and it looks more readable
 ` className={twMerge('relative text-L text-primary', className)}`

Reply:
fixed